test/raven.utils.js

```diff
@@ -97,6 +97,45 @@ describe('raven.utils', function () {
         lol: 1
       }], callback);
     });
+
+    it('should extract context from last stack line', function (done) {
+      var parseStack = raven.utils.parseStack;
+      var callback = function (frames) {
+        var frame = frames.pop();
+        frame.pre_context.should.be.an.instanceOf(Array);
+        frame.context_line.should.be.type('string');
+        frame.context_line.trim().should.endWith('undeclared_function();');
+        frame.post_context.should.be.an.instanceOf(Array);
+        done();
```

Comment:
Can we add an assertion here on what `frame.in_app` should be, and add a comment to the effect of part 4 of [this comment](https://github.com/getsentry/raven-node/pull/118#r31550565) explaining what's going on in this test? It's not initially obvious that this tests anything not existing before this PR.